ghproxy/README.md

```diff
@@ -3,4 +3,18 @@
 ghProxy is a reverse proxy HTTP cache optimized for use with the GitHub API (https://api.github.com).
 It is essentially just a reverse proxy wrapper around [ghCache](/ghproxy/ghcache) with some additional prometheus instrumentation logic to monitor disk usage and push metrics to a prometheus push gateway.
 
-ghProxy is designed to reduce API token usage by allowing many components to share a single ghCache. Note that components must use the same API token to benefit from the cache and avoid clobbering existing cache entries for other tokens.
\ No newline at end of file
+ghProxy is designed to reduce API token usage by allowing many components to share a single ghCache. Note that components must use the same API token to benefit from the cache and avoid clobbering existing cache entries for other tokens.
+
+## How to use
+
+To use ghProxy for your prow instance,
```

Comment:
Instead of words here, why not 

```
kubectl apply -f prow/cluster/gce-ssd-retain_storageclass.yaml
kubectl apply -f prow/cluster/ghproxy_deployment.yaml
kuebctl apply -f prow/cluster/ghproxy_service.yaml
```